cclib/io/ccio.py

```diff
@@ -292,6 +293,8 @@ def ccopen(source, *args, **kargs):
         else:
             inputfile.seek(0, 0)
         if not is_stream:
+            if filetype == Turbomole:
+                source = sort_turbomole_outputs(source)
```

Comment:
@berquist A simple check `if is_listofstrings:` will remove the second failure where `job.last` is passed as the first parameter. Despite that I am a bit conufesd on how to make cclib guess that a file is a part of Turbomole outputs. When we use `ccopen()`  with a list as input then we are able to guess the correct parser since `job.last` is needed to find the correct trigger  but `ccget` seems to be looking at one file at a time and that is throwing it off since it only correctly predicts for `job.last` and not for the files (e.g. `basis`, `control`).

Reply:
As Eric alluded, we need to treat all the files as one buffered file object. This has come up in the past, and we've always come to the conclusion that we want to hide the particulars of looping over multiple files, streams or whatnot, and expose a consistent buffered file object to the parsers. We actually have a `FileWrapper` class in `logfileparser.py` which IIRC was meant to implement this kind of thing, so if it doesn't we need to extend it.